src/users/users.service.ts

```diff
@@ -62,7 +62,7 @@ export class UsersService {
         
         // if (store.length > 0) 
         //     user.manager = await this.usersRepository.findOne({where:{employeeId : 1}}) // set employeeId 1 as manager for now
-        user.manager = createuserDto.manager
+        user.manager = await this.usersRepository.findOne({where:{employeeId : createuserDto.managerId}})
```

Comment:
For the future, when we consider multiple companies, do we want to have the manager be identified by a userId and companyId?

Reply:
yes, create user needs some improvements to be more robust and to handle the case of multiple companies, for now every user created should be part of the same company based on our logic here and lack of a create company endpoint (Assuming you're not manually entering data into the database)